bee-inx/src/block.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_block as bee;
+use inx::proto;
+
+/// The [`Block`] type.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Block {
+    /// The [`BlockId`](bee::BlockId) of the block.
+    pub block_id: bee::BlockId,
```

Comment:
Maybe just rename this field to `id` because there's no other `id` in this struct to distinct it from.

Reply:
I'm following the naming conventions from the [protobuf definitions](https://github.com/iotaledger/inx/blob/develop/proto/inx.proto) here.